src/server/server.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include "mir/server.h"
 
+#include "frontend_wayland/xdg_activation_v1.h"
```

Comment:
This looks like a wrong approach.

We shouldn't be exposing the whole (or, probably, any) of xdg-activation to the server code: 

1. it is pointless to "pretend" to be a client requesting a token, as the pretend client will not have focus when the token is invoked;
2. this code shouldn't care when xdg-activation-v2 or xdg-activation (stable) comes along

A better approach would be to have a token issue/validation API used by both the launch code and the activation protocol